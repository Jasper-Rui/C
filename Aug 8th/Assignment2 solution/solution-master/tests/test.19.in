TRX 1 A012345678901234567890 B012345678901234567890 150000 20
TRX 2 B012345678901234567890 C012345678901234567890 150000 20
TRX 3 C012345678901234567890 D012345678901234567890 150000 20
TRX 4 D012345678901234567890 E012345678901234567890 150000 20
TRX 5 E012345678901234567890 F012345678901234567890 150000 20
TRX 6 F012345678901234567890 G012345678901234567890 150000 20
TRX 7 G012345678901234567890 H012345678901234567890 150000 20
TRX 8 H012345678901234567890 I0123456789012345678901 150000 20
MINE
MINE
MINE
END

