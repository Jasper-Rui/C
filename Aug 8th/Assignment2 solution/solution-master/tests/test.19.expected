Adding transaction: 1 A012345678901234567890 B012345678901234567890 150000 20
Adding transaction: 2 B012345678901234567890 C012345678901234567890 150000 20
Adding transaction: 3 C012345678901234567890 D012345678901234567890 150000 20
Adding transaction: 4 D012345678901234567890 E012345678901234567890 150000 20
Adding transaction: 5 E012345678901234567890 F012345678901234567890 150000 20
Adding transaction: 6 F012345678901234567890 G012345678901234567890 150000 20
Adding transaction: 7 G012345678901234567890 H012345678901234567890 150000 20
Adding transaction: 8 H012345678901234567890 I0123456789012345678901 150000 20
Block mined: 1 0 0x00000000 4
1 A012345678901234567890 B012345678901234567890 150000 20
2 B012345678901234567890 C012345678901234567890 150000 20
3 C012345678901234567890 D012345678901234567890 150000 20
4 D012345678901234567890 E012345678901234567890 150000 20
0x0000024c 0x0076bd3a
Block mined: 2 1 0x0076bd3a 3
5 E012345678901234567890 F012345678901234567890 150000 20
6 F012345678901234567890 G012345678901234567890 150000 20
7 G012345678901234567890 H012345678901234567890 150000 20
0x00000028 0x000f5f93
Block mined: 3 2 0x000f5f93 1
8 H012345678901234567890 I0123456789012345678901 150000 20
0x000000b2 0x00a96248
